build.gradle

```diff
@@ -106,8 +104,11 @@ task functional(type: Test) {
     javaexec {
       main = "uk.gov.hmcts.reform.managecase.befta.ManageCaseAssignmentBeftaMain"
       classpath += sourceSets.functionalTest.runtimeClasspath + sourceSets.main.output + sourceSets.test.output
-      args = ['--plugin', "json:${cucumberReports.outputDir}/cucumber.json", '--tags', 'not @Ignore', '--glue',
-              'uk.gov.hmcts.befta.player', 'src/functionalTest/resources/features']
+      args = ['--threads', '5',
```

Comment:
We have very long wait times before cases appear in ES results. Could 10 threads not be better?

Reply:
As [before](https://github.com/hmcts/aac-manage-case-assignment/pull/150#discussion_r529625394):
> The **5 threads & 8 second wait is a compromise to avoid 'Case not found' Elastic Search issues in local development** when running FTA tests locally against MCA running at console or IDE and CCD in docker (4 CPU, 10GB + 2 GB swap, with active compose files: `backend sidam sidam-local sidam-local-ccd elasticsearch frontend`).
> 
> I tried the same 10 threads as the other repos and with the original 5 seconds wait however as ElasticSearch is used throughout MCA's APIs local failures were too frequent.   So after trying many different combinations.  5+8 was the quickest complete time to successfully run current FTAs (201, 202, 203 + 206) more often than hitting a 'Case not found' failure, **although it still often does**.